.eslintrc.prod.json

```diff
@@ -0,0 +1,15 @@
+{
+    "extends": "./.eslintrc.json",
+    "rules": {
+        "iot-cardboard-js/debug-logging-disabled": "error",
+        "no-debugger": "error",
+        "no-multiple-empty-lines": "error",
+        "react-hooks/exhaustive-deps": "off",
```

Comment:
so when it is prod, these rules will be overridden with those values? but how would it help us in dev as we send PRs?

Reply:
"prod" here is just the CI pipeline. This is the file that is used anytime you're not local.